### PR TITLE
Adds a sound effect to health analyzers

### PR DESCRIPTION
**Pull request by nmajask**
## About the PR 
Adds sound to health analyzers and PDAs when they finish a health scan. Has code for playing a sound when starting or ending a scan though only the latter is used. Tested it with the regular analyzer, PDA, and giga-cancer health analyzer. Sound effect is from this PR: https://github.com/Baystation12/Baystation12/pull/25396

**Media**

https://user-images.githubusercontent.com/14363906/211697445-d5053d54-ac85-4358-9d03-139068ec3f50.mp4

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: a sound effect to health analyzers on a successful scan

### CODE REVIEW

File: Content.Server/Medical/HealthAnalyzerSystem.cs [hunk 1]
```diff
@@ -69,6 +77,13 @@ private void OnAfterInteract(EntityUid uid, HealthAnalyzerComponent healthAnalyz
         private void OnTargetScanSuccessful(TargetScanSuccessfulEvent args)
         {
             args.Component.CancelToken = null;
+
+
```
Comment (0x6273) [inline Content.Server/Medical/HealthAnalyzerSystem.cs:81]:
```suggestion

```
One too many blank lines.

Comment (nmajask) [reply Content.Server/Medical/HealthAnalyzerSystem.cs]: removed it in the latest commit

**Comment by nmajask** [comment]
> The null checks are redundant because `PlayPvs` takes a nullable and does nothing if it's null.

Thanks for pointing that out, I just copied the code from the medical stacks so I didn't know if it was needed. I removed it in the latest commit. Should probably go back and do the same for the medical stacks if its also redundant, but that's probably best left to another PR.

File: Resources/Audio/Items/Medical/license.txt [hunk 2]
```diff
@@ -1,10 +1,11 @@
 The following sounds were used from freesound:
- 
+
 herbertboland - heavyskirtmovement.wav - CC BY 3.0
 vinrax - cloth-sounds.mp3 - CC BY-NC 3.0
-lordvanye - skintouching.wav - CC0 1.0 
+lordvanye - skintouching.wav - CC0 1.0
+FM Synthesis - scanner blip.wav - CC BY 4.0
```
Comment (0x6273) [inline Resources/Audio/Items/Medical/license.txt:6]: The file is called "healthscanner.ogg" and not "scanner blip.wav"

**Review by 0x6273 — APPROVED** [review]

**Pull request merged**